receiver/sqlserverreceiver/scraper.go

```diff
@@ -538,6 +538,7 @@ func (s *sqlServerScraperHelper) recordDatabaseStatusMetrics(ctx context.Context
 	const dbSuspect = "db_suspect"
 	const dbOffline = "db_offline"
 	const cpuCount = "cpu_count"
+	const computerUptime = "computer_uptime"
```

Comment:
nit: could we unify the usage of "computer uptime" and "uptime computer" in this PR for consistency?

Reply:
I'll rename the metric to `os.uptime`, if that works:
1. Per [naming conventions](https://github.com/open-telemetry/semantic-conventions/blob/main/docs/general/naming.md#system-specific-attributes), the system name should come first.
2. We already use os as a component, which is similar in meaning to computer.

Reply:
done!